hoomd/md/compute.py

```diff
@@ -282,84 +283,71 @@ def num_particles(self):
             return None
 
 
-class thermoHMA(Compute):
+class ThermoHMA(Compute):
     R""" Compute HMA thermodynamic properties of a group of particles.
 
     Args:
-        group (``hoomd.group``): Group to compute thermodynamic properties for.
+        filter (``hoomd.filter``): Particle filter to compute thermodynamic
+            properties for.
         temperature (float): Temperature
```

Comment:
We should probably mention this should be the same as the temperature of the integrator methods used for this group.